server/build-plugins.js

```diff
@@ -53,8 +55,10 @@ async function installRemotePackages(installPackages) {
 in node_modules. Try deleting the node_modules directory and running `npm install` again.'
     );
     logger.error(e.stack);
+    await logger.close();
     process.exit(1);
   }
```

Comment:
I think that we could get away with a `finally` clause here so that we only have to `await logger.close()` once

Reply:
good call